dojo/tools/anchore_grype/parser.py

```diff
@@ -70,3 +168,20 @@ def _convert_severity(self, val):
             return "Info"
         else:
             return val.title()
+
+    def get_cve(self, vuln_id, rel_id):
+        if vuln_id and vuln_id.startswith('CVE'):
+            return vuln_id
+        elif rel_id and rel_id.startswith('CVE'):
+            return rel_id
+        else:
+            return vuln_id
+
+    def get_cvss(self, cvss):
+        if cvss:
+            for cvss_item in cvss:
+                if cvss_item['version'].startswith('3'):
```

Comment:
Please consider using the package `cvss` to be aligned with other parsers.
https://defectdojo.github.io/django-DefectDojo/contributing/how-to-write-a-parser/#do-not-parse-cvss-by-hand-vector-score-or-severity

Reply:
The `cvss` item is a list of CVSS2 and CVSS3 entries. The parser needs to find the CVSS3 entry. I don't see how the cvss library helps to find this entry. We can test all cvss vectors with the cvss library, but just using the version number as provided in the json seems to be easier.

Reply:
Using `cvss` module provide us a check of the vectors out-of-the-box which means that the parser could be able to see not-well formatted vectors when reading the report and not crash the upper layers (which check this data with a regex in Django model).

Reply:
done